app/controllers/messages_controller.rb

```diff
@@ -13,30 +18,33 @@ def index
 	#Displays all messages for a specific country
 	def show
 		@messages = Message.where(recipient: params[:id]).order('created_at DESC')
-		
+
 		respond_to do |format|
 			format.html
 			format.json { render json: @messages }
 		end
 	end
 
-
 	def new
 		@message = Message.new
 	end
 
-
 	def create
```

Comment:
This function is still the same as the API one, as it may prove useful to Control(?)